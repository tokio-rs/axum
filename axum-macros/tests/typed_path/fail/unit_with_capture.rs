use axum_macros::TypedPath;
use serde::Deserialize;

#[derive(TypedPath, Deserialize)]
#[typed_path("/users/:id")]
struct MyPath;

fn main() {}
