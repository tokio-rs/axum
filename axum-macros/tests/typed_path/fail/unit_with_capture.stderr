error: Typed paths for unit structs cannot contain captures
 --> tests/typed_path/fail/unit_with_capture.rs:5:14
  |
5 | #[typed_path("/users/:id")]
  |              ^^^^^^^^^^^^
